extensions/purelymail/src/check-account-credit.tsx

```diff
@@ -0,0 +1,14 @@
+import { showToast, Toast } from "@raycast/api";
+import { getAccountCredit } from "./utils/api";
+import { Response } from "./utils/types";
+import { showFailureToast } from "@raycast/utils";
+
+export default async function CheckAccountCredit() {
+  const response: Response = await getAccountCredit();
+
+  if (response.type === "error") {
+    await showFailureToast(response.message, { title: response.code });
+  } else {
+    await showToast(Toast.Style.Success, "SUCCESS", `Credit: ${response.result.credit}`);
```

Comment:
I would use `updateCommandMetadata` instead, and maybe add an interval to the call so it updates, for example, every hour.

Reply:
https://github.com/raycast/extensions/assets/30526133/06d0e80c-b83f-4830-b266-7f25f4ef97da

Hi 👋 .

This is a great suggestion! I have implemented as shown in attached video.

Best 👍🏽 .

Reply:
Do you think it makes sense to round the number, there is a lot of decimals going on there 🙂

Reply:
It's a reasonable suggestion which I've implemented and also updated the metadata screenshot accordingly;

<img width="1000" alt="purelymail-5-check-account-credit" src="https://github.com/raycast/extensions/assets/30526133/a072cb41-c021-4be5-bda2-d35351772a30">